app/addons/documents/mango/components/ExecutionStats.js

```diff
@@ -0,0 +1,127 @@
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
```

Comment:
can you rename to make this file all lowercase

Reply:
the other component files are camelcased - shouldn't we stick to convention?

Reply:
Yes. We should have made them all lowercase. Lets leave it as is then. And we can lowercase them all in another PR